.github/workflows/release-periodically.yml.j2

```diff
@@ -1,4 +1,3 @@
-{% if distro_release == "rawhide" %}
 name: Release periodically
 # Create a GitHub release on schedule
```

Comment:
Maybe please add a comment that it's kept on the fedora branches with only manual trigger to make things easier. The comment is no longer the whole truth :(